src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -5,6 +5,7 @@
 using System;
```

Comment:
This file changed substantially as it moved from using inheritance to control behaviors to a composable plugin system. It may be easiest to just review the new contents vs. the diff.